### ISSUE TITLE
The sc_ally util stopped working

### ISSUE THREAD

**Issue by dario-ramos** [dario-ramos/baires-scp#16]
In my attempt to test sc_ally, I learned the following:

The sc_ally util needs to connect to a scamper instance that is running in daemon mode, listening for commands on an http port. This can easily be done in a separate terminal. For example, to listen on port 31337:

`sudo ./scamper/scamper -P 31337`

Once scamper is running in the background, sc_ally can be called. It needs a file as input, and that file should contain at least 2 IP addresses per line. For example:

[input.txt]
```
192.0.2.1 192.0.32.10 192.0.31.60
192.0.2.2 192.0.31.8
192.0.2.3 192.0.30.64

```

With that file created in the same directory as the sc_ally binary (for simplicity), it's possible to run sc_ally:

```
dario@dario-desktop:~/Documents/eclipse-workspace/baires-scp/utils/sc_ally$ ./sc_ally -a input.txt -o outfile.warts -p 31337 -w 5 -t log.txt
1688663631: p 1, w 0, v 0 : ping -P udp-dport -i 1 -c 7 -o 5 8.8.8.8
```

The problem when it comes to automatically testing this is that it never exits. I'm not sure if this is by design, or I am missing something. I tried both public and private addresses, setting a timeout, and setting a debug log file, which doesn't give any extra information. It just replicates the only output line printed.

Also, when I interrupt sc_ally via Ctrl+C, the scamper daemon process crashes with a segmentation fault.

**Comment by dario-ramos** [comment]
I tried running this in another PC, using a clean scamper install, with the same version this code base is based on (20191102). With that version, everything worked fine: sc_ally printed some extra output, and eventually exited. Also, interrupting sc_ally did not crash scamper.
Maybe having the thesis work PC be connected to Internet via USB tethering is causing these problems? I'll open another issue for tracking this.

**Comment by dario-ramos** [comment]
Fixed in PR #18
